core/src/main/java/nl/nn/adapterframework/soap/SoapWrapperPipe.java

```diff
@@ -202,47 +202,45 @@ public void stop() {
 		}
 	}
 
-    @Override
-	public PipeRunResult doPipe(Object input, IPipeLineSession session) throws PipeRunException {
+	@Override
+	public PipeRunResult doPipe(Message message, IPipeLineSession session) throws PipeRunException {
 		String result;
 		try {
 			if ("wrap".equalsIgnoreCase(getDirection())) {
-				String payload = input.toString();
+				String payload = message.asString();
```

Comment:
Waarom een string? alles moet gewoon message blijven

Reply:
maar hij word in een reeks ifjes doorgegeven. 
Moet uiteindelijk met zoiets als een reeks XmlFilters opgelost worden.